src/Redux/Admins/actions.js

```diff
@@ -17,6 +17,13 @@ import {
   RESET_STATE
 } from './constants';
 
+export const setDefaultAdmin = (admin) => {
+  return {
+    type: 'SET_DEFAULT_ADMIN',
```

Comment:
could be added to constants.js file and imported from there
```suggestion
    type: SET_DEFAULT_ADMIN,
```